File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/PlaceBuyNowServerRequest.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

class PlaceBuyNowServerRequest
{
    /**
     * The ID of the buy-it-now server configuration to order. Use the server listing from &#x60;GET /servers/order/buy_now_server&#x60; to find valid IDs.
     */
    #[DTA\Data(field: "server_id", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "float"])]
    public float|null $server_id = null;

    /**
     * The fully-qualified hostname to assign to the server.
     */
    #[DTA\Data(field: "server_hostname", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $server_hostname = null;

    /**
     * The root or administrator password to set on the server.
     */
    #[DTA\Data(field: "server_root_password", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $server_root_password = null;

}

File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/PlaceBuyNowServerRequest.scala

package org.openapitools.client.model


case class PlaceBuyNowServerRequest (
    /* The ID of the buy-it-now server configuration to order. Use the server listing from `GET /servers/order/buy_now_server` to find valid IDs. */
    _server_id: Option[Number],
    /* The fully-qualified hostname to assign to the server. */
    _server_hostname: Option[String],
    /* The root or administrator password to set on the server. */
    _server_root_password: Option[String]
)
object PlaceBuyNowServerRequest {
    def toStringBody(var_server_id: Object, var_server_hostname: Object, var_server_root_password: Object) =
        s"""
        | {
        | "server_id":$var_server_id,"server_hostname":$var_server_hostname,"server_root_password":$var_server_root_password
        | }
        """.stripMargin
}
